integrations/apollo/src/index.ts

```diff
@@ -24,7 +25,8 @@ export type NhostApolloClientOptions = {
   connectToDevTools?: boolean
   cache?: InMemoryCache
   onError?: RequestHandler
```

Comment:
```suggestion
  /**
   * @deprecated Please use `generateLinks` instead.
   */
  onError?: RequestHandler
```

Please note that depending on the result of the open discussion, we might want to change the deprecation message.